README-Milestone4.md

```diff
@@ -0,0 +1,63 @@
+# Milestone 4 - Collaborate on a Shared Uni
+In this section, you will collaborate on a shared Uni
+
+## Collaborate Using the GraphQL Explorer
+
+Using the GraphQL Explorer, each Participant should add a new Hero or Villain to the shared Uni created in the previous Milestone 
+
+```bash
+
+mutation addHero($description:String,$name:String, $slug:String,$username:String) {
```

Comment:
Since we're using the GraphQL Explorer please include the variable declaration JSON or just put in static values.